action.yml

```diff
@@ -92,8 +92,8 @@ runs:
           "api_key": "${{ inputs.api_key }}",
           "test": { "fmf": {
               "url": "${{ inputs.tmt_repository }}",
-              "ref": "${{ inputs.tmt_ref }}",
-              "name": "{{ inputs.tests_tmt_ref }}",
+              "ref": "${{ inputs.tmt_branch }}",
+              "name": "{{ inputs.test_fmf_plan }}",
```

Comment:
In our action we have it as `tests_regex` https://github.com/oamg/testing-farm-service-action/blob/main/action.yml#L18 , because that's how we are mainly using it.

Reply:
yeah, it would be good to agree here on something we can then fix in `v0.2` of our API, the naming also in API was not very  good

Reply:
I will open another PR fixing some stuff I see

Reply:
@thrix have you managed to think of something we should change yet?

Reply:
put my ideas on :spiral_notepad:  in https://github.com/sclorg/testing-farm-as-github-action/pull/21

Reply:
@fernflower tests_regex is a bit misleading as it is really a regex for the plans. But yeah, you can write tests in plans also :/

Reply:
Then maybe `plans_regex`? Imho it's important to show that more than 1 plan can be matched